packages/mask/src/social-network-adaptor/twitter.com/collecting/getSearchedKeyword.ts

```diff
@@ -20,7 +20,7 @@ export default function getSearchedKeywordAtTwitter(): string {
     if (!isTwitter(activatedSocialNetworkUI)) return ''
     const params = new URLSearchParams(location.search)
     const hashTagMatched = location.pathname.match(/\/hashtag\/([A-Za-z]+)/)
-    if (location.pathname === '/search' && !params.get('f')) return decodeURIComponent(params.get('q') ?? '')
+    if (location.pathname === '/search') return params.get('q') ?? ''
```

Comment:
We should not display `SearchBar` if the URL hits `https://twitter.com/search?q=test&src=typed_query&f=image`.

Reply:
fixed